src/org/traccar/protocol/Tk103ProtocolDecoder.java

```diff
@@ -111,38 +132,140 @@ private void decodeType(Position position, String type, String data) {
             case "BO01":
                 position.set(Position.KEY_ALARM, decodeAlarm(data.charAt(0) - '0'));
                 break;
+            // Motion Alert with Location
```

Comment:
Remove all unnecessary comments like this.

Reply:
done